### DISCUSSION TITLE
It all works - web interface times out over night

### DISCUSSION THREAD

**Discussion by sjmgeezer** [Security-Onion-Solutions/securityonion#4575] [Unsupported Versions] [answered]
Can I stop the web interface from timing out - needs a auto refresh or 'mouse jiggler' like app.
Not the machine as other screens continue to function - not SecOnion?

Chrome OS / Browser
Dell laptop

System up and running aOK - just the timeout - needs logging back next day.
Making sense?

Any help / feedback = awesome.

Thanks - hopefully so very simple/obvious

**Comment by jertel** [comment, marked as answer]
The current login session timeout is 24h. However, we'll get this customizable in an upcoming release for those who wish to change it to a timeout shorter, or longer than the default.

See #4586
